src/amqp.rs

```diff
@@ -0,0 +1,68 @@
+use lapin::tcp::{OwnedIdentity, OwnedTLSConfig};
+use vector_config::configurable_component;
+
+/// Connection options for AMQP.
+#[configurable_component]
+#[derive(Clone, Debug)]
+pub(crate) struct AMQPConfig {
+    /// URI for the AMQP server.
+    ///
+    /// Format: amqp://user:password@host:port/vhost?timeout=seconds
```

Comment:
```suggestion
    /// Format: amqp://<user>:<password>@<host>:<port>/<vhost>?timeout=<seconds>
```
Does this make it more obvious those sections should be replaced by the operator?